src/libcollections/borrow.rs

```diff
@@ -14,6 +14,7 @@
 
 use core::clone::Clone;
 use core::cmp::{Eq, Ord, Ordering, PartialEq, PartialOrd};
+use core::convert::As;
```

Comment:
Could these traits also go into the libcore prelude? I think files like this can have `use core::prelude::*` at the top as well (slowly been migrating over time).